docs/source/toolbar.rst

```diff
@@ -0,0 +1,99 @@
+LUCID Toolbar Configuration
+===========================
+
+``LUCID`` can be launched with a toolbar file that specifies
+the tabs that are displayed below the device grid.
+These tabs can be configured to hold buttons
+that open other screens.
+
+Invocation
+----------
+
+The invocation to include this toolbar file is:
+
+.. code-block:: bash
+
+    lucid --toolbar /path/to/toolbar/file.yaml HUTCHNAME
+
+Options and Limitations
+-----------------------
+
+- The user can specify an arbitrary number of tabs.
+- Each tab can have an arbitrary number of columns.
+- The columns are filled from left to right, then from top to bottom.
+- Only ``PyDMShellCommand`` and ``PyDMRelatedDisplayButton`` widgets are supported.
+- Any available pyqt property or general python attribute can be specified on either
+  of these button types.
+
+Guidelines
+----------
+
+- The related display option (``display``) should be used whenever it is possible to do so,
+  because the screen will open more quickly compared to a shell script, which will need to
+  initialize an entire new python shell with new imports.
+
+File Format
+-----------
+
+The toolbar configuration is stored in a yaml file that encodes a dictionary.
+Each key in this top-level dictionary is the name of a tab to include.
+These tabs will be arranged in file order from left to right.
+
+Each of these tab names should contain another dictionary with the following top-level keys:
+
+- ``config`` (optional)
+- ``buttons``
+
+The only config option right now is ``cols``, which defaults to 4.
```

Comment:
```suggestion
The only ``config`` option right now is ``cols``, which defaults to 4.
```
My eyes totally skipped over this when looking for what the ``config`` option does, maybe highlighting it would make it more obvious?